packages/devp2p/src/dpt/server.ts

```diff
@@ -255,9 +255,8 @@ export class Server extends EventEmitter {
    * @param msg Message text to debug
    */
   private debug(messageName: string, msg: string) {
-    debug(msg)
     if (this.msgDebuggers[messageName]) {
       this.msgDebuggers[messageName](msg)
-    }
+    } else debug(msg)
```

Comment:
```suggestion
    } else {
      debug(msg)
```

Reply:
ah sorry nvm already saw holger mentioned this in his comment below https://github.com/ethereumjs/ethereumjs-monorepo/pull/1643/files#r786632639